### PR TITLE
Not re-fetch ramCPEntry via CP index

### PR DESCRIPTION
**Pull request by JasonFengJ9**
Not re-fetch `ramCPEntry` via `CP` index

Use `ramCPEntry` directly instead of re-fetching it via `CP` index. This matches the code pattern for `invokePrivate` in similar case.
Also refactoring code format within the the method affected.

This is a following-up cleanup after https://github.com/eclipse/openj9/pull/5694.

Reviewer: @gacholio 
FYI: @DanHeidinga 

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: runtime/vm/resolvesupport.cpp [hunk 1]
```diff
@@ -1367,13 +1367,13 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
 		 * and storing the result into the cpClass->methodTypes table.
 		 */
 		if ((NULL != cpShapeDescription)
-		&& (resolvedClass == J9VMJAVALANGINVOKEMETHODHANDLE(vm))
+			&& (resolvedClass == J9VMJAVALANGINVOKEMETHODHANDLE(vm))
```
Comment (DanHeidinga) [inline runtime/vm/resolvesupport.cpp]: I'd prefer that these spacing changes not be made.  The formatting without it clear that the the if statement continues as a blcok

File: runtime/vm/resolvesupport.cpp [hunk 2]
```diff
@@ -1367,13 +1367,13 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
...
 		) {
 			J9UTF8 *nameUTF = J9ROMNAMEANDSIGNATURE_NAME(nameAndSig);
 			J9UTF8 *sigUTF = J9ROMNAMEANDSIGNATURE_SIGNATURE(nameAndSig);
 			if ((J9CPTYPE_HANDLE_METHOD == J9_CP_TYPE(cpShapeDescription, cpIndex))
-			|| J9UTF8_LITERAL_EQUALS(J9UTF8_DATA(nameUTF), J9UTF8_LENGTH(nameUTF), "invokeExact")
-			|| J9UTF8_LITERAL_EQUALS(J9UTF8_DATA(nameUTF), J9UTF8_LENGTH(nameUTF), "invoke")
+				|| J9UTF8_LITERAL_EQUALS(J9UTF8_DATA(nameUTF), J9UTF8_LENGTH(nameUTF), "invokeExact")
+				|| J9UTF8_LITERAL_EQUALS(J9UTF8_DATA(nameUTF), J9UTF8_LENGTH(nameUTF), "invoke")
```
Comment (DanHeidinga) [inline runtime/vm/resolvesupport.cpp]: same here

File: runtime/vm/resolvesupport.cpp [hunk 3]
```diff
@@ -1442,81 +1442,81 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
...
 				}
 				break;
 			default:
 				if (J9UTF8_LITERAL_EQUALS(initialMethodName, initialMethodNameLength, "compareAndSet")
-				 || J9UTF8_LITERAL_EQUALS(initialMethodName, initialMethodNameLength, "getAndBitwiseOr")
-				 || J9UTF8_LITERAL_EQUALS(initialMethodName, initialMethodNameLength, "weakCompareAndSet")
-				 || J9UTF8_LITERAL_EQUALS(initialMethodName, initialMethodNameLength, "compareAndExchange")
+					|| J9UTF8_LITERAL_EQUALS(initialMethodName, initialMethodNameLength, "getAndBitwiseOr")
+					|| J9UTF8_LITERAL_EQUALS(initialMethodName, initialMethodNameLength, "weakCompareAndSet")
+					|| J9UTF8_LITERAL_EQUALS(initialMethodName, initialMethodNameLength, "compareAndExchange")
```
Comment (DanHeidinga) [inline runtime/vm/resolvesupport.cpp]: all through here

Comment (JasonFengJ9) [reply runtime/vm/resolvesupport.cpp]: All these space changes have been removed. Please have another look.

File: runtime/vm/resolvesupport.cpp [hunk 4]
```diff
@@ -1607,21 +1607,21 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
...
 			/* Only allow non-interface method to call invokePrivate, private interface method should use "invokeInterface" bytecode
 			 * The else case will throw ICCE for private interface method 
 			 */
 			if (J9_ARE_ALL_BITS_SET(romMethod->modifiers, J9AccPrivate) && J9_ARE_NO_BITS_SET(resolvedClass->romClass->modifiers, J9AccInterface)) {
 				/* Private method found, will not be in vTable, point vTable index to invokePrivate */
-				if (ramCPEntry != NULL) {
-					ramCPEntry->method = method;
+				if (NULL != ramCPEntry) {
 					UDATA methodIndexAndArgCount = J9VTABLE_INVOKE_PRIVATE_OFFSET << 8;
+					ramCPEntry->method = method;
```
Comment (DanHeidinga) [inline runtime/vm/resolvesupport.cpp]: Why reorder this line?

Comment (JasonFengJ9) [reply runtime/vm/resolvesupport.cpp]: The intention was to declare variables at top of the scope though I understood that is required for `C` code only.
Restored the order.

**Comment by DanHeidinga** [comment]
Jenkins test sanity plinux,win jdk8

**Comment by DanHeidinga** [comment]
Jenkins test sanity xlinux,zlinux jdk11

**Comment by DanHeidinga** [comment]
Jenkins test extended zlinux,aix jdk12

**Comment by JasonFengJ9** [comment]
`Test_openjdk8_j9_sanity.functional_x86-64_windows_Personal ` failed due to a unrelated exception:
```
Caused: java.io.IOException: Remote call on win2012r2-x86-2 failed
	at hudson.remoting.Channel.call(Channel.java:963)
	at hudson.Launcher$RemoteLauncher.launch(Launcher.java:1059)
```

**Pull request merged**